src/vm/amd64/profiler.cpp

```diff
@@ -41,6 +41,11 @@ typedef struct _PROFILE_PLATFORM_SPECIFIC_DATA
     UINT64      flt5;
     UINT64      flt6;
     UINT64      flt7;
+    UINT64      rdi;
```

Comment:
may be we can use r12,r13 for it? something like this
mov [rbp-dispr12], r12
mov [rbp-dispr13] r13
mov r12, rsi
mov r13, rdi
...
call PROFILE_ENTER 

and inside PROFILE_ENTER  save r12&r13 ?